src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -5578,7 +5565,7 @@ private static async Task<string> GetLatestPackageVersion(string packageIndexUrl
 
         public static string GetAzureFeedForVersion(string version)
         {
-            return version.StartsWith("7.0") || version.StartsWith("8.0")
```

Comment:
It shouldn't matter, does it?

If it does should it then be based on the channel instead of the version? This way a "current" would always use the default feed, but "edge" would go on the dotnetbuilds one.

Reply:
I reverted this change after I made the other and it still didn't work as it couldn't find the files to download them. I looked at changing it to use the channel instead of the version, but it seems to be quite a viral change.

I've pushed that up into another branch in this commit. If you think that's a better change I can add it to this PR. https://github.com/martincostello/crank/commit/bb0261bd282017eb2ecccb5861e11aa6ae4daf41